docs-src/concepts/what-is-a-worker-identity.md

```diff
@@ -0,0 +1,53 @@
+---
+id: what-is-a-worker-identity
+title: What is a Worker Identity?
+sidebar_label: Worker Identity
+description: A Worker Identity is an identifier for a Worker Client.
+tags:
+  - term
+  - explanation
+---
+
+Workers have an associated identifier that helps identify the specific Worker instance.
```

Comment:
When we say "Worker instance", Is this at the Worker Process level or the Worker Entity level?

From below it seems like it is at the Process level - 
If so, it means one can have multiple Entities in the same process, but they would be identified as the same.

Can we clarify that?

Reply:
In Core-based SDKs, `identity` can be specified per Client or per Worker (I mean instances of the SDK's Worker class), while in Go and Java, it can only be specified on Client. To keep documentation simple, it's probably better not include per-worker suffix in snippets.